NeoML/src/TraditionalML/WordDictionary.cpp

```diff
@@ -96,10 +92,12 @@ void CWordDictionary::Finalize( long long minCount )
 {
 	words.QuickSort<Descending<CWordWithCount>>();
 
-	const CWordWithCount countSeparator{ "", minCount - 1 };
-	const int insertionPoint = words.FindInsertionPoint<Descending<CWordWithCount>>( countSeparator );
+	if( minCount > INT64_MIN ) {

```

Comment:
Unnecessary in case of non-negative frequencies.
But check for `minCount` positivity will be useful.